modules/apps/foundation/captcha/captcha-taglib/src/main/java/com/liferay/captcha/taglib/internal/servlet/ServletContextUtil.java

```diff
@@ -14,25 +14,42 @@
 
 package com.liferay.captcha.taglib.internal.servlet;
 
+import com.liferay.captcha.configuration.CaptchaConfiguration;
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+
+import java.util.Map;
+
 import javax.servlet.ServletContext;
 
 import org.osgi.service.component.annotations.Activate;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
 import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Pei-Jung Lan
  */
-@Component(immediate = true)
+@Component(
+	configurationPid = "com.liferay.captcha.configuration.CaptchaConfiguration",
+	immediate = true
+)
 public class ServletContextUtil {
 
+	public static final CaptchaConfiguration getCaptchaConfiguration() {
+		return _instance._captchaConfiguration;
```

Comment:
_captcaConfiguration is a non-static variable. We cant access non-static variable from a static method so we need to get it from _instance